### PR TITLE
Canned Response Actions

### PR DESCRIPTION
**Pull request by agreenspan24**
## Description

Question Responses/Interaction Steps have a useful feature called actions that use extensions to integrate specific responses with an outside source or trigger a workflow. This work makes the same feature available on Canned Responses. 

Originally, I built this feature with multiple actions per canned response, thinking that in an NGP VAN action, the admin might want the response to sync to both a canvass result and a survey response or activist code. However, I have now learned that survey responses and activist codes automatically apply a canvass result of Canvassed. In addition, none of our admins in Colorado felt the need for more than one action per response. Any further engagement could be solved with tags, which have a many to one relationship with canned responses.

List
Edit

# Checklist:

- [x] I have manually tested my changes on desktop and mobile
- [x] The test suite passes locally with my changes
- [x] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [x] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
- [x] I have made any necessary changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

**Comment by Frydafly** [comment]
Does this resolve issue #1645?

**Comment by agreenspan24** [comment]
@Frydafly this isn't related to that issue, but I think that issue should be closed. You can already add multiple tags to canned responses now.

File: src/extensions/action-handlers/complex-test-action.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export async function available(organizationId) {
 // What happens when a texter saves the answer that triggers the action
 // This is presumably the meat of the action
 export async function processAction({
-  interactionStep,
+  actionObject,
```
Comment (schuyler1d) [inline src/extensions/action-handlers/complex-test-action.js:45]: please update `mobilecommons-signup.js` to remove interactionStep as well

Comment (agreenspan24) [reply src/extensions/action-handlers/complex-test-action.js:45]: @schuyler1d done! Let me know if there's anything else you need. 

Does this mean we're moving forward with my canned response work?

Comment (agreenspan24) [reply src/extensions/action-handlers/complex-test-action.js:45]: Also, we'll have some merge issues with the new Zapier action: https://github.com/MoveOnOrg/Spoke/pull/1995

CC: @crayolakat

**Comment by crayolakat** [comment]
Hey @agreenspan24 , code looks good to me! I wanted to do some local testing as well. Can you resolve the merge conflicts first?

**Comment by agreenspan24** [comment]
> Hey @agreenspan24 , code looks good to me! I wanted to do some local testing as well. Can you resolve the merge conflicts first?

Hey @crayolakat! I think this is already included and merged in this PR - https://github.com/MoveOnOrg/Spoke/pull/2037. We combined them since there would be merge conflicts. I'll close this one out.

**Pull request closed without merging**